gulp/tasks/css.js

```diff
@@ -0,0 +1,42 @@
+// task to build scss
+module.exports = function () {
+  $.gulp.task("css", function () {
+    return $.gulp
+      .src($.path.src.css)
+      .pipe($.plugins.plumber())
+      .pipe($.plugins.sourcemaps.init())
+      .pipe(
+        $.plugins
+          .sass({
+            includePaths: $.sassPaths,
+            outputStyle: "compressed",
+          })
+          .on(
+            "error",
+            $.plugins.notify.onError("SCSS-Error: <%= error.message %>")
+          )
+      )
+      .pipe($.plugins.groupCssMediaQueries())
+      .pipe(
+        $.plugins.autoprefixer({
+          overrideBrowserslist: ["last 5 version"],
+          cascade: true,
+        })
+      )
+      .pipe(
+        $.plugins.uncss({
+          html: ["src/**/*.html"],
```

Comment:
Please use `$.path.src.html` instead of the string

Reply:
fixed: $ .path.src.html is specified instead of the string